src/main/java/com/zufar/onlinestore/product/dto/ProductInfoRequestResponseDto.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.product.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.util.UUID;
+
+public record ProductInfoRequestResponseDto(
+
+        @NotNull(message = "Id is the mandatory attribute")
```

Comment:
why do you need validation for the dto class which is used as response object, not as a request object?
it doesn't make any sense
delete all these validation annotations, please
don't forger to format this class

Reply:
This class will be use for Request too. For example - in getById or more MB.

Reply:
Fixed. We will use for Response ONLY.